demo/lib/vm.bash

```diff
@@ -161,7 +161,23 @@ vm-install-cri-resmgr() {
         vm-command "go get -d -v github.com/intel/cri-resource-manager"
         CRI_RESMGR_SOURCE_DIR=$(awk '/package.*cri-resource-manager/{print $NF}' <<< "$COMMAND_OUTPUT")
         vm-command "cd $CRI_RESMGR_SOURCE_DIR && make install && cd -"
-    else
+    elif [ "$binsrc" == "packages/debian-10" ]; then
```

Comment:
I'd suggest already now using something like `[  "${binsrc#packaging/}" != "$binsrc" ]` as a catch-all case for all current and future packaging tests.

Reply:
Changed this to matching "package/{ubuntu,debian}"*, because the branch is written for debs only.